templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -148,7 +151,7 @@ contents:
         
       env ETCDCTL_API=3 ${ETCDCTL} snapshot restore $SNAPSHOT_FILE \
         --name $ETCD_NAME \
-        --initial-cluster ${ETCD_NAME}=https://${ETCD_DNS_NAME}:2380 \
+        --initial-cluster ${ETCD_CONNSTRING} \
         --initial-cluster-token etcd-cluster-1 \
         --skip-hash-check=true \
         --initial-advertise-peer-urls https://${ETCD_IPV4_ADDRESS}:2380 \
```

Comment:
Since we are here we should really make this a param as well for flexability?

 if [ -z "${ETCD_INIT_PEERS_URLS}" ]; then
        ETCD_INITIAL_PEERS="https://${ETCD_IPV4_ADDRESS}:2380
 fi

Reply:
Good catch, I'll address that in the next PR and make sure CI uses this param